Dockerfile

```diff
@@ -178,9 +179,10 @@ ADD http://www.random.org/strings/?num=10&len=8&digits=on&upperalpha=on&loweralp
 
 WORKDIR /
 
+
 RUN echo "===> Install Elpis"
 # Remove `--single-branch` and replace with `--branch <your_branch_name>` below for development
-RUN git clone --single-branch --depth=1 https://github.com/CoEDL/elpis.git
+RUN git clone --single-branch --branch ben-deps-update --depth=1 https://github.com/CoEDL/elpis.git
```

Comment:
You'll want this in master after the merge right?

Reply:
nope, i'll remove that when the PR is ready

Reply:
Looks like I'll have to include Jinja in this first round.. more commits coming...